CMakeLists.txt

```diff
@@ -64,8 +64,13 @@ endif()
 # Included stuff
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 
-
 set(IRRLICHTMT_BUILD_DIR "" CACHE PATH "Path to IrrlichtMt build directory.")
+
+
+if(MINGW AND CMAKE_HOST_SYSTEM_NAME MATCHES "Windows" AND IRRLICHTMT_BUILD_DIR STREQUAL "")
+set(IRRLICHTMT_BUILD_DIR "C:/Program Files (x86)/Irrlicht/lib/cmake/IrrlichtMt")
```

Comment:
This seems pretty arbitrary to me.

Reply:
But every condition is necessary

And there is no value in taking these three judgment conditions apart

Reply:
Do you have any better suggestions for revision

Reply:
> This seems pretty arbitrary to me.

It is arbitrary. The `Program Files (x86)` would make me assume that this will only work when compiling a 32-bit version of Irrlicht on a 64-bit install of Windows.

Reply:
But in fact, after many incomplete tests, this is the path automatically installed after compiling irrlichtmt in both 64 bit and 32-bit msys environments